apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/explorer/ecms-explorer.less

```diff
@@ -714,8 +714,9 @@
 			textarea {
 				font-size: 13px;
 				height: 175px;
-				width: 90%;
+				width: 95%;
```

Comment:
Should we keep 90% ? as there is not problem on the witdh, i think we should keep this unmodified
So for consistency, we should use also 90% in the other less file